WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1803,3 +1805,12 @@ FileTypeCal         = @FileIconCommonStyle@ background-color:#94385d;
 FileTypeVCard       = @FileIconCommonStyle@ background-color:#666666;
 FileTypeMedia       = @FileIconCommonStyle@ background-color:#9e61ca;
 FileTypeMail        = @FileIconCommonStyle@ background-color:@PrimaryBrandingColor@
+
+#######################################################################
+#   Create Contact-Group Dialog
+#######################################################################
+
+CreateContactGroupDialog        =
```

Comment:
Suggestion: 
We do have some dialog related styling near line 1316 starting from "DialogBodyText" . 
We can have common styling section for Dialogs maybe? Not specific to create contact like in this case?